Create cookie-attributes.md (#8503)
auth0/docs: articles/sessions/concepts/cookie-attributes.md

Aside from cookies set by the browser’s javascript context, these options are provided by the server with its HTTP responses as part of the set-cookie headers, the browser, upon receipt of the responses parses these and maintains its cookie jar according to them.

## `sameSite` attributes changes

Effective around February 2020, Google Chrome (v80) will change its cookie handling. Two things are changing with this planned release:

* Cookies set without `sameSite` attribute will be defaulted to `lax` instead of the current implicit value `none`.

* Cookies set with `sameSite=none` must also be secure or they’ll be rejected to be saved in the browser’s cookie jar. 

The reason for this change is to improve default security and prevent CSRF attacks. 

### Changes Auth0 has made

Auth0 has made changes to the following cookies: 

* `auth0`: User session
* `auth0-mf`: Multi-Factor information
* `did`: Device/User agent identifier

For all the cookies: 
* We will set the `sameSite=None` attribute and the cookie will always require to use HTTPS regardless of the environment.

> * We will set a fallback cookie `*_compat` for legacy browsers.

Reviewer: Since this is a public document I would be more specific
```suggestion
* We will set fallback cookies for each cookie to be used by legacy browsers that do not support `sameSite=none`. The name of the new cookies are `auth0_compat`, `auth0-mf_compat` and `did_compat`.
```

### Changes You Need to Make
To be ready for this change, you should ensure the following: 

* Review the list of [currently unsupported browsers](https://www.chromium.org/updates/same-site/incompatible-client.)

> * If you use `form_post`, you must also use `sameSite=none` running on `https://` - no exceptions are being made by Chrome for `localhost`.

Reviewer: ```suggestion
* If your application uses `response_mode=form_post` when interacting with Auth0, you must also use `sameSite=none` running on `https://` - no exceptions are being made by Chrome for `localhost`.
```

![Fresh Interaction](/media/articles/sessions/cookie-return-interaction.png)

The following table describes the `sameSite` attribute values.

| Attribute | Description |
| -- | -- |
| `strict` | Only send the cookie if the user is navigating within the website origin bounds it came from. |
| `lax` | Only send the cookie if the user is navigating within the website origin bounds it came from or if he’s being redirected to it. |
| None | (No sameSite attribute provided option) Unless other conditions (such as other options or  third-party cookies blocked) disqualify the cookie from being sent. |

## Affects 

The table below shows how the `sameSite` attribute changes may affect your apps.

| App Behavior | Affected by Change |
| -- | -- |
| Cookies set as `sameSite=nome` while the website is not `https://` | Yes |
| Cookies don't have explicit `sameSite` attribute value set and are required in a cross-origin context (such as HTTP Form Post, embedding an iframe) | Yes |
| Native apps (everything not cookies + web based) | No (M2M) |

> | Already setting an excplicit `sameSite` cookie attribute value | No |

Reviewer: ```suggestion
| Already setting an explicit `sameSite` cookie attribute value | No |
```
